.github/workflows/ci.yml

```diff
@@ -7,11 +7,41 @@ concurrency:
 
 jobs:
 
+  build:
```

Comment:
While I see this speeds up builds, this skips building on windows (and mac), which is something we'll want to check as well.

Reply:
Note to self - it appears that the lerna issue is from the fact that there is a file called `lerna.js` in the root and so on windows `lerna ...` trys to open that file rather than run the `lerna` command.